File: Documentation/API/Parsers/Midi/ReadResolutionFromMidiData.js
import { readFileSync } from 'node:fs';

import RhythmGameUtilities from '@rhythm-game-utilities/core';

const data = readFileSync('./tests/Mocks/song.mid');

const resolution = RhythmGameUtilities.ReadResolutionFromMidiData(data);

console.log(resolution); // 480

File: Documentation/API/Parsers/Midi/ReadResolutionFromMidiData.lua
---@type RhythmGameUtilities
local rhythmgameutilities = require("rhythmgameutilities")

local file = io.open("./tests/Mocks/song.mid", "rb")

if file then
    local data = file:read("*a")

    local value = rhythmgameutilities.read_resolution_from_midi_data(data);

    print(value) -- 480
end
